platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -154,12 +129,12 @@ private CRL createCRL(String location) {
     }
 
     /**
-     * Abstracted for unit tests to allow test class to set a specific property location
+     * Loads the properties from a given location.
      *
      * @param location location of properties file
-     * @return Properties from
+     * @return Properties from the file
      */
-    Properties loadProperties(String location) {
+    public Properties loadProperties(String location) {
```

Comment:
It was previously called externally in the CrlInterceptorTest class. It's only being called in the CrlCheckerTest class now though, so I'll bump it down to package-private.